### ISSUE TITLE
Why are <li> being used as the clickable items

### ISSUE THREAD

**Issue by harryhaigh** [reactjs/react-tabs#361]
Isn't it best practice to use<buttons> instead of <li> as the clickable elements for tabs.  If this library is focused on accessibility then why isn't it following best practices?

**Comment by AgmarTorres** [comment]
I agree, maybe we can refactore this part with a different way to do a clickable tag.
